src/playground/Pages/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { AreaChartWidget } from "@/components/Widgets/Charts/AreaChartWidget"
```

Comment:
I can't comment on the file name, but should this be called `Overview` for consistency, perhaps?

Reply:
This one is outdated